### DISCUSSION TITLE
Single Quotes vs Double Quotes in Filters

### DISCUSSION THREAD

**Discussion by ZorzStudios** [FreshRSS/FreshRSS#7163] [Q&A] [answered]
Hello! Just installed FreshRSS and learning my way around its excitement. :)

In the [article on filters](https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field), I see that both single quotes vs double quotes are used, e.g. 'composed name' and "my label". Is the choice between them as strict as exemplified? Meaning, it would fail if I use single quotes for 'my label' and double for "composed name"?

Thank you!

**Comment by Alkarex** [comment, marked as answer]
Double quotes are also fine for quoting labels. The main rule is that you must use the same type of quote to open and close.

**Reply by ZorzStudios** [reply]
Oh, that's curious; thank you for the clarification! :)

**Comment by ZorzStudios** [comment]
On a related note, it's a bit unclear to me if I'm allowed to place filter criteria on **one line**.

The app itself clearly instructs to use separate lines, but the help article gives an example like:
label:label, label:"my label"

By the way, if we search by a keyword as free-text, will it look for it in both title _and_ the body? I don't see a designated in-body search criteria.

Thank you for your support, and happy new year!!

**Reply by Alkarex** [reply]
See a bit further down in the documentation https://freshrss.github.io/FreshRSS/en/users/10_filter.html#with-the-search-field
about combining several search criteria.

When on the same line, the following will search for titles which contain "Hello" and contain "World" as well:
`intitle:Hello intitle:World`

When on two lines, the following will search for titles which contain "Hello" as well as for titles containing "World", independently:
```
intitle:Hello
intitle:World
```

Correct, there is no operator for searching only in body, but I believe we have a ticket somewhere to add it.

**Reply by ZorzStudios** [reply]
Wonderful, this is clearer now. I had read the referenced article very closely, but the way you now illustrate the difference between single and separate lines is better. If I may, I suggest adding such an illustration to the article, too. Thank you!

**Reply by Alkarex** [reply]
PR welcome
* https://github.com/FreshRSS/FreshRSS/blob/edge/docs/en/users/10_filter.md
* https://github.com/FreshRSS/FreshRSS/blob/edge/docs/fr/users/03_Main_view.md
